examples/prod_pdc.rb

```diff
@@ -7,8 +7,13 @@ def main
     config.log_level = :debug
   end
 
-  releases = PDC::V1::Release.all!.to_a
-  ap releases
+  #releases = PDC::V1::Release.all!.to_a
+  #ap releases
+
+  ap PDC::V1::ReleaseVariant.where(:release => 'ceph-1.3@rhel-7', :uid => 'Calamari').first
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)